### PR TITLE
diamond: import from homebrew/science

### PR DESCRIPTION
**Pull request by tmozgach**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by sjackman** [comment]
Thanks, Tanya!

File: Formula/diamond.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class Diamond < Formula
+  desc "Accelerated BLAST compatible local sequence aligner"
+  homepage "https://ab.inf.uni-tuebingen.de/software/diamond/"
+
```
Comment (ilovezfs) [inline Formula/diamond.rb]: The blank line should be removed.

File: Formula/diamond.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    assert_match "gapextend", shell_output("#{bin}/diamond help 2>&1")
```
Comment (ilovezfs) [inline Formula/diamond.rb]: We'll need a test that exercises and verifies the functionality of the software.

**Review by ilovezfs — DISMISSED** [review]

**Comment by tmozgach** [comment]
@ilovezfs good?

File: Formula/diamond.rb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      fflrrslalsprlecsgaisahcklrlpgsrhspasasrvagttgarhharlifvflvet
+      gfhrvsqdgldlltsXsarlglpkcwdyrrepprpa
+      >gnl|alu|HSU14568_Alu_Sb_consensus_rf-2
+      ffXdgvslcrpgwsavarsrltassasrvhaillpqppeXlglqapattpgXflyfXXrr
+      gftvlarmvsisXprdppasasqsagitgvshrar
+      >gnl|alu|HSU14568_Alu_Sb_consensus_rf-3
+      ffetesrsvaqagvqwrdlgslqapppgftpfsclslpsswdyrrppprpanfcifsrdg
+      vspcXpgwsrspdlvirpprppkvlglqaXatapg
+    EOS
+    assert_match "Processed 6 sequences, 572 letters.", shell_output("#{bin}/diamond makedb --in nr.faa -d nr")
```
Comment (ilovezfs) [inline Formula/diamond.rb]: output = shell_output("#{bin}/diamond makedb --in nr.faa -d nr")
assert_match "Processed 6 sequences, 572 letters.", output

**Review by ilovezfs — APPROVED** [review]
minor nit but :+1:

**Comment by ilovezfs** [comment]
Thanks again @tmozgach! :gem:

**Comment by sjackman** [comment]
Thanks, Tanya! And thanks for merging, @ilovezfs!

**Pull request closed without merging**